src/mon/Monitor.cc

```diff
@@ -464,6 +464,7 @@ const char** Monitor::get_tracked_conf_keys() const
     "clog_to_graylog",
     "clog_to_graylog_host",
     "clog_to_graylog_port",
+    "host",
```

Comment:
@tchaikov @xiexingguo I get the point of the commit, was merely looking for an explanation to a change that's not at all obvious in the given patch set.